webrecorder/web-replay-gen
# Enable GitHub Pages by default

(https://github.com/webrecorder/web-replay-gen/issues/17) Enables GH pages by default. Pushes to `main` will automatically build the site as long as `publish-gh-pages.yml` is present.

Updates to README: https://github.com/webrecorder/web-replay-gen/blob/github-pages-workflow/README.md#web-replay-gen

### Manual testing
In your own repo created from this template repo:
```
git remote add template git@github.com:webrecorder/web-replay-gen.git
git fetch template
git cherry-pick 0cc80e..f479533
git push
```

Verify that GH pages builds as expected.

Tested with https://github.com/SuaYoo/web-replay-gen-sample (https://suayoo.github.io/web-replay-gen-sample)
